doc/nrf/libraries/caf/settings_loader.rst

```diff
@@ -45,6 +45,12 @@ To use the module, you must complete the following steps:
 This function is called on the settings loader module initialization.
 After each of modules that sets bit in :c:func:`get_req_modules` is initialized, the |settings_loader| calls :c:func:`settings_load` function and starts loading all the settings from non-volatile memory.
 
+File system as settings backend
+===============================
+
+If a file system is used as settings backend (:kconfig:option:`CONFIG_SETTINGS_FS`), application must make sure that the file system is mounted before Zephyr settings subsystem is initialized.
+The CAF settings loader module calls :c:func:`settings_subsys_init` initialization function during system boot with ``APPLICATION`` level and initialization priority of :kconfig:option:`CONFIG_APPLICATION_INIT_PRIORITY`.
```

Comment:
```suggestion
If the settings backend (:kconfig:option:`CONFIG_SETTINGS_FS`) is a file system, application must make sure that the file system is mounted before the Zephyr settings subsystem is initialized.
The CAF settings loader module calls the :c:func:`settings_subsys_init` initialization function during system boot with ``APPLICATION`` level and initialization priority of :kconfig:option:`CONFIG_APPLICATION_INIT_PRIORITY`.
```

Reply:
`If the settings backend is a file system (:kconfig:option:`CONFIG_SETTINGS_FS`) ...`
I would prefer to order it like that, because `CONFIG_SETTINGS_FS` is used to select file system backend for the settings